frontend/packages/dev-console/src/actions/add-resources.tsx

```diff
@@ -60,7 +61,14 @@ export const resolvedURLWithParams = (
 };
 
 export const AddActions: { [name: string]: ActionFactory } = {
-  FromGit: (namespace, application, contextSource, path, accessReviewDisabled) => ({
+  FromGit: (
+    namespace,
+    application,
+    contextSource,
+    path,
+    accessReviewDisabled,
+    isActionTypeEnabled,
```

Comment:
Instead of adding more and more parameters to this "helpers", could we just skip calling `FromGit` (and all others) if this parameter is false?